src/host/pico_platform/include/pico/platform.h

```diff
@@ -47,7 +47,7 @@ extern void tight_loop_contents();
 #define __STRING(x) #x
 #endif
 
-#ifndef _MSC_VER
+#if !defined(_MSC_VER) || defined(__clang__)
```

Comment:
did you mean `if !defined(_MSC_VER) && !defined(__clang__)`?

Reply:
No, it's not. The changes were intended to disable LL.74-103 when using clang-x86_64-pc-windows-msvc.

clang-x86_64-pc-windows-msvc has built-in functions like gcc syntax. Specifically there are `__builtin_unreachable` and `__builtin_clz`. Also, clang-x86_64-pc-windows-msvc has a macro _MSC_VER already defined. So, LL.74-103 were enabled. As a result, the compilation fails.